flake.nix

```diff
@@ -0,0 +1,123 @@
+{
+  description =
+    "tinygrad: For something between PyTorch and karpathy/micrograd. Maintained by tiny corp.";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+
+    flake-utils = {
+      url = "github:numtide/flake-utils";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs = { self, nixpkgs, flake-utils }@I:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        P = import nixpkgs {
+          inherit system;
+          config.allowUnfree = true;
+        };
+        B = builtins;
+        L = P.lib;
+
+        pythonVersion = "310";
+        pythonPackageStr = "python${pythonVersion}";
+        pythonNixpkgs = P."${pythonPackageStr}Packages";
+
+        tinygrad = pythonNixpkgs.buildPythonPackage {
+          pname = "tinygrad";
+          version = "v.0.6.0";
```

Comment:
Is the `.` here between `v` and `0` intentional?

Reply:
Nope just a typo: good catch, I'l fix it.